src/rest-server/deploy/legacy_user_migrate.py

```diff
@@ -19,14 +19,18 @@ def __init__(self, user_name):
 
 class TransferClient:
 
-    def __init__(self, etcd_uri, k8s_uri):
+    def __init__(self, etcd_uri, k8s_uri, admin_groupname):
         self.etcd_uri = etcd_uri
         self.k8s_uri = k8s_uri
+        self.admin_group = admin_groupname
         self.etcd_conn = http.client.HTTPConnection(self.etcd_uri)
         self.k8s_conn = http.client.HTTPConnection(self.k8s_uri)
         self.flag_path = '/v2/keys/transferFlag'
         self.etcd_prefix = '/users/'
         self.secret_ns = "pai-user"
+        self.secret_ns_user_V2 = "pai-user-v2"
+        self.secret_ns_group_v2 = "pai-group"
```

Comment:
Please uniform word cases. V2 should be lower case.
secret_ns_group_v2 should be secret_ns_group

Reply:
Done